Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -294,8 +294,10 @@ class SearchSettingsTableViewController: ThemedTableViewController {
             let index = indexPath.item + 1
             let engine = model.orderedEngines[index]
 
-            model.deleteCustomEngine(engine) {
+            model.deleteCustomEngine(engine) {[weak self] in
```

Comment:
```suggestion
            model.deleteCustomEngine(engine) { [weak self] in
```
spacing. LGTM otherwise

Reply:
@adudenamedruby I am an beginner in Open Source. Just had one doubt do I need to raise the PR again?